CHANGELOG.md

```diff
@@ -9,10 +9,16 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 
-* **Footer** now accepts a `version` prop to include your app version, and 
-  a `versionUrl` prop if you want the version number to be a link.  Works
+* **Footer** now accepts a `version` prop to include your app version, and
+  a `versionUrl` prop if you want the version number to be a link. Works
   great if you use dotenv (see **Footer** docs for more)
 
+### Changed
+
+* Reformat code according to StandardJS.
+* Removed unused prop in **NavTab** component. This is a non-breaking change
```

Comment:
@DanielBaird

Reply:
Ahh i guess i should have reviewed the doc changes more closely :) Okay it's all good.